sgt-back-end/index.js

```diff
@@ -0,0 +1,154 @@
+const pg = require('pg');
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const express = require('express');
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `
+    select "gradeId",
+           "name",
+           "course",
+           "score",
+           "createdAt"
+      from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      if (!grades) {
+        res.status(404).json({
+          error: 'Cannot find grades'
+        });
+      } else {
+        res.json(grades);
+      }
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error occurred.'
+      });
+    });
+});
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+app.post('/api/grades', (req, res) => {
+  const bodyObject = req.body;
+  const name = bodyObject.name;
+  const course = bodyObject.course;
+  const score = bodyObject.score;
+
+  if (!name || !course || !score || !Number.isInteger(score) || score < 0 || score > 100) {
+    res.sendStatus(400);
+    return;
+  }
```

Comment:
`400 Bad Request` responses should include a description of how the request is invalid.